vite.config.js

```diff
@@ -1,14 +1,20 @@
 import { defineConfig } from "vite";
 import react from "@vitejs/plugin-react";
 import dotenv from "dotenv";
+import eslintPlugin from "vite-plugin-eslint";
 
 dotenv.config();
 
 export default defineConfig({
   base: "",
   root: "",
   envPrefix: "RECORD_MANAGER_",
-  plugins: [react()],
+  plugins: [
+    react(),
+    eslintPlugin({
+      cache: false, // disable eslint cache to avoid conflicts
```

Comment:
why do we need it?

Reply:
to ensure that eslint performs linting checks without relying on cached results